main.py

```diff
@@ -0,0 +1,215 @@
+"""
+    To add categories print 1
+    If you want to see the list print 2
+                    Group by:
+    To group by month you need to print 3
+    To group by category you need to print 4
+    If you want to group by month and by category print 5
+    To show your expenses as a chart print 6
+    To create a file with your expenses print 7
+    To stop interacting with data print 8
+"""
+
+import csv
+from datetime import datetime
+import re
+import matplotlib.pyplot as plt
+
+fieldnames = ["Category", "Date", "Amount"]
+categories = {}
+
+
+def check_int(prompt):
+    """
+    Checking the correctness of the number input
+    """
+    while True:
```

Comment:
I see that u imported `re` libary. use it then :)

Reply:
re library will be used for validation